tools/scripts/build/download-trending.ts

```diff
@@ -0,0 +1,48 @@
+import { writeFileSync } from 'fs';
+import fetch from 'node-fetch';
+import yaml from 'js-yaml';
+import { trendingSchemaValidator } from '../../../client/i18n/schema/trendingSchema';
+
+import envData from '../../../config/env.json';
+const { clientLocale } = envData;
+
+const cdnUrlCreator = (lang: string) =>
+  `https://cdn.freecodecamp.org/universal/trending/${lang}.yaml`;
+
+(async () => {
```

Comment:
```suggestion
const download async () => {
```
We don't need to use process.exit if we go this way